README.md

```diff
@@ -83,6 +97,12 @@ Conductor can be ran via the docker image, and it can even be ran via `npx` for
 npx @graphql-conductor/bin ./conductor.config.yaml
```

Comment:
maybe we should also point out above that `config.json` is the default one, and it should be supplied as a first argument otherwise.

Reply:
added there: https://github.com/the-guild-org/conductor/pull/567/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R131